docs/tutorial/access/client.md

```diff
@@ -351,6 +358,8 @@ To start using EESSI just [source the initialisation script](../eessi/using-eess
 source /cvmfs/software.eessi.io/versions/2023.06/init/bash
 ```
 
+!!! danger "update link to latest version? maybe add to eessi a symlink /cvmfs/software.eessi.io/versions/latest/init/bash" 
```

Comment:
```suggestion
```

For now this is still the only version we had. We actually used to have a `latest` symlink, but we stopped doing that because figuring out when to update the target of that is ... challenging.

Reply:
Sounds reasonable. And for the tutorial even an old version would work.